osu.Game/Screens/OnlinePlay/Playlists/AddPlaylistToCollectionButton.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using System.Threading.Tasks;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions;
+using osu.Game.Collections;
+using osu.Game.Database;
+using osu.Game.Graphics;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Graphics.UserInterfaceV2;
+using osu.Game.Online.Rooms;
+using osu.Game.Overlays;
+using osu.Game.Overlays.Notifications;
+
+namespace osu.Game.Screens.OnlinePlay.Playlists
+{
+    public partial class AddPlaylistToCollectionButton : RoundedButton
+    {
+        private readonly Room room;
+
+        private LoadingLayer loading = null!;
+
+        [Resolved]
+        private RealmAccess realmAccess { get; set; } = null!;
+
+        [Resolved]
+        private BeatmapLookupCache beatmapLookupCache { get; set; } = null!;
+
+        [Resolved(canBeNull: true)]
+        private INotificationOverlay? notifications { get; set; }
+
+        public AddPlaylistToCollectionButton(Room room)
+        {
+            this.room = room;
+            Text = "Add Maps to Collection";
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
+            BackgroundColour = colours.Gray5;
```

Comment:
Not sure where this colour choice came from, but feels wrong. Best just use a default button colour instead.